continuous-integration/setup/setup_build_and_test.sh

```diff
@@ -17,10 +17,15 @@ apt-yes install \
     clinfo \
     wget \
     python3-pip \
-    python-catkin-tools \
     unzip \
     || exit $?
 
+if [[ "$UBUNTU_VERSION" == "20.04" ]]; then
+    apt-yes install python3-catkin-tools python3-osrf-pycommon || exit $?
```

Comment:
I guess `python3-osrf-pycommon` became a dependency in U20/Neotic?

Reply:
This is taken from https://answers.ros.org/question/353113/catkin-build-in-ubuntu-2004-noetic/ and is needed for some reason.